README.md

```diff
@@ -115,7 +116,7 @@ complete beginners.
 
 ## Extras : Books and Video Courses! :
 
-<img src="https://cdn-icons-png.flaticon.com/128/3730/3730087.png" height="60px"> <img src="https://image.flaticon.com/icons/png/128/2096/2096018.png" height="60px">
```

Comment:
reviewer comment. Only one image exists. Removing the 2nd one fixes the heading position.

Reply:
Yes, that would be good @cadamini . 
Thanks.